test/e2e/service_cleanup.py

```diff
@@ -86,19 +86,22 @@ def service_cleanup(config: dict):
     )
 
     try:
-        delete_sns_topic(resources.SnsTopicARN)
+        delete_sns_topic(resources.SnsTopic1)
+        delete_sns_topic(resources.SnsTopic2)
     except:
-        logging.exception(f"Unable to delete SNS topic {resources.SnsTopicARN}")
+        logging.exception(f"Unable to delete SNS topics {resources.SnsTopic1}, {resources.SnsTopic2}")
```

Comment:
Please have `try: except:` block for each individual `delete_sns_topic` call.
This will ensure that `SnsTopic2` delele is tries even if `SnsTopic1` delete gets exception.